File: joseffe-barroso/Python/aula04/while/exercise43.py
sequence = int(input("Qual é o valor do termo?"))

while (sequence <= 0) or (sequence > 50):
    sequence = int(input("O valor da sequência de termos deve ser positivo e menor que cinquenta. Informe novamente: "))

numerator = 1
denominator = 2
index = 1
sumNumbers = 0

while (index < sequence):
    numerator = (index ** 2) + 1
    denominator = (index ** 3)
    divisao = numerator / denominator
    if (index == 1): print(f'{numerator}')
    else: print(f"{numerator}/{denominator}")
    index = index + 1
    sumNumbers = sumNumbers + divisao

print('A soma é: %.2f' %sumNumbers)

File: joseffe-barroso/Java/src/entrada_processamento_saida/aula04/While/Exercise43.java
import java.util.Scanner;

public class Exercise43 {

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);

		int sequence, index;
		double numerator, denominator, divisao, sumNumbers;
		
		System.out.println("Qual é o valor da sequencia?");
		sequence = scanner.nextInt();

		while (sequence <= 0 || sequence > 100) {
			System.out.println("O valor da sequência deve ser positivo e menor que cem. Informe novamente:  ");
			sequence = scanner.nextInt();
		}

		numerator = 1;
		denominator = 2;
		index = 1;
		sumNumbers = 0;

		while (index < sequence){
		    numerator = Math.pow(index, 2) + 1;
		    denominator = Math.pow(index, 3);
		    divisao = numerator / denominator;
		    if (index == 1) System.out.printf("%f", numerator);
		    else System.out.printf("%f/%f", numerator, denominator);
		    index = index + 1;
		    sumNumbers = sumNumbers + divisao;
		}
			
		scanner.close();
	}

}
